src/sandbox/patchers/interval.ts

```diff
@@ -7,14 +7,17 @@
 import { noop } from 'lodash';
 
 const rawWindowInterval = window.setInterval;
-const rawWindowClearInterval = window.clearInterval;
+// 兼容ie10、11下，非全局clearInterval,报调用的对象无效，
+// @ts-ignore
+window.rawWindowClearInterval = window.clearInterval;
 
 export default function patch(global: Window) {
   let intervals: number[] = [];
 
   global.clearInterval = (intervalId: number) => {
     intervals = intervals.filter((id) => id !== intervalId);
-    return rawWindowClearInterval(intervalId);
+    // @ts-ignore
+    return window.rawWindowClearInterval(intervalId);
```

Comment:
I guess use `rawWindowClearInterval.call(window, intervalId)` could solve the compatible issue, rather than bind a new global variable on window.

Reply:
> I guess use `rawWindowClearInterval.call(window, intervalId)` could solve the compatible issue, rather than bind a new global variable on window.

@kuitos  you are right.   `rawWindowClearInterval.call(window, intervalId)` works too.

Reply:
> I guess use `rawWindowClearInterval.call(window, intervalId)` could solve the compatible issue, rather than bind a new global variable on window.

I fix it